RELEASE.md

```diff
@@ -6,6 +6,12 @@ Please follow the established format:
 - Include the ID number for the related PR (or PRs) in parentheses
 -->
 
+# Release x.x.x
+
+## Bug fixes and other changes
+
+- Ensure URL parameters and parameter nodes are in sync. (#1449)
```

Comment:
should we add another release note that says something like 

Ensure filter states for task, data, and parameter nodes as set by the user, even after browser refresh.